setup.cfg

```diff
@@ -38,3 +38,9 @@ ignore_errors = true
 
 [mypy]
 ignore_missing_imports = True
+disallow_any_unimported = True
+disallow_incomplete_defs = True
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+disallow_untyped_decorators = True
+no_implicit_optional = True
```

Comment:
I use these options with reference to some famous OSS, such as below.

https://github.com/dropbox/mypy-protobuf/blob/master/mypy.ini

Reply:
Please add a new line at EOF

Reply:
fixed here

2ac17a4